ansible/roles/test/tasks/continuous_link_flap.yml

```diff
@@ -0,0 +1,196 @@
+# This is continuous link flap test. In this test,
+# 1.) Flap all interfaces one by one in 1-3 iteration
+# to cause BGP Flaps.
+# 2.) Flap all interfaces on peer (FanOutLeaf) one by one 1-3 iteration
+# to cause BGP Flaps.
+# 3.) Watch for memory (show system-memory) ,orchagent CPU Utilization
+# and Redis_memory.
+# Pass Criteria: All routes must be re-learned with < 5% increase in Redis
+# and ORCH agent CPU consumption below 10% after 3 mins after stopping flaps.
+
+- block:
+    - name: Record  memory status at start
+      shell: show system-memory
+      register: memory_output
+
+    - debug: msg="Memory Status {{memory_output.stdout}}"
+
+    - name: Record Redis Memory at start
+      shell: redis-cli info memory | grep used_memory_human | sed -e 's/.*:\(.*\)M/\1/'
+      register: start_time_redis_memory
+
+    - debug: msg="Redis Memory {{start_time_redis_memory.stdout}} M"
+
+# track only ipv4 routes. IPv6 addresses are removed with ifconfig
+# down command, which may bring ipv6 BGP neighborship down. This
+# must be addressed in dev code first.
```

Comment:
Could you align comment with below block? Many more occurrence below.

Reply:
Sure.